helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -60,11 +60,8 @@ spec:
             resources: {{- toYaml .Values.head.resources | nindent 14 }}
             securityContext:
             {{- toYaml .Values.head.securityContext | nindent 14 }}
+            {{- with concat .Values.common.containerEnv .Values.head.containerEnv }}
```

Comment:
Does the `concat` handle duplications?

Reply:
`concat` just concatenate an arbitrary number of lists into one:
https://helm.sh/docs/chart_template_guide/function_list/#concat

The current implementation doesn't handle duplication so it would be handled exactly the same way.